lib/msal-browser/test/encode/Base64Decode.spec.ts

```diff
@@ -95,6 +95,25 @@ describe("Base64Decode.ts Unit Tests", () => {
             );
         });
 
+        it("Decode idtokenclaims with special characters", () => {
```

Comment:
There's a bunch of [localization test cases](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/d1bbf52b5c3ab14e456af076a8b4b1e7c5c024c1/lib/msal-node/test/crypto/CryptoProvider.spec.ts#L49) in msal-node. I'm surprised we don't have these in msal-browser as well. Would be good to get these added here too

Reply:
Added one more case in browser localization tests